examples/basic.rs

```diff
@@ -0,0 +1,54 @@
+use std::{collections::HashSet, path::Path};
+
+use bpf_linker::{Cpu, Linker, LinkerInput, LinkerOptions, OptLevel, OutputType};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let path = Path::new("/path/to/object-or-bitcode");
```

Comment:
wouldn't this example be better as Examples in the rustdocs for the respective methods?